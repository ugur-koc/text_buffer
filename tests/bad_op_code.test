bod_op_code args
exit