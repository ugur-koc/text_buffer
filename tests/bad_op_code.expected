Available Operations: insert append eraseAt erase replace load save help exit 
Text Buffer: 
Unknown operation code!

Text Buffer: 
